src/TrustedShops.php

```diff
@@ -225,10 +225,19 @@ public static function wp_footer() {
 
     $products_sku = [];
     if ($product->get_type() === 'variable') {
-      $variations = $product->get_available_variations();
-      foreach ($variations as $variation) {
-        $products_sku[] = $variation['sku'] ?? '';
-      }
+      // Gets variations skus, avoiding "get_available_variations()".
+      global $wpdb;
+      $variation_ids = $product->get_visible_children();
+      $placeholders = implode(',', array_fill(0, count($variation_ids), '%d'));
+      $products_sku = $wpdb->get_col(
+        $wpdb->prepare(
+          "SELECT pm.meta_value AS attachment_id
+          FROM wp_posts p
+          INNER JOIN wp_postmeta pm ON pm.post_id = p.ID AND pm.meta_key = '_sku'
+          WHERE p.ID IN ($placeholders)",
+          $variation_ids
+        )
+      );
```

Comment:
Not sure why you are joining on posts here… - am I missing something?

And what's `attachment_id` doing here? ;-)

Please also follow the following code style:
```suggestion
      $products_sku = $wpdb->get_col($wpdb->prepare("
        SELECT pm.meta_value AS sku
        FROM wp_postmeta pm
        WHERE pm.post_id IN ($placeholders) AND pm.meta_key = '_sku'
      ", $variation_ids));
```

Reply:
I must have been really tired :), thanks.